stats.tex

```diff
@@ -0,0 +1,50 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
```

Comment:
But the integrals/sums are automatically evaluated. From the end user point of view, E(X) returns the expectation, etc... I would like to leave it as it is for the sake of simplicity, If you agree.